src/ui/reportView/callReport.ts

```diff
@@ -160,17 +161,31 @@ function createCommand(
  */
 async function runVisualizeCommand(command: string, harnessName: string): Promise<reportMetadata> {
 	try {
+		// Get the full resolved path for the root directory of the crate
+		const directory = path.resolve(getRootDir());
+		const commmandSplit: CommandArgs = splitCommand(command);
+
+		// Get cargo command and args for the command to be executed
+		const args = commmandSplit.args;
+
+		const options = {
+			shell: false,
+			cwd: directory,
+		};
+
+		const kaniBinaryPath = await getKaniPath('cargo-kani');
 		vscode.window.showInformationMessage(`Generating viewer report for ${harnessName}`);
 		vscode.window.showWarningMessage(warningMessage);
-		const { stdout, stderr } = await execPromise(command);
+		const { stdout, stderr } = await execPromise(kaniBinaryPath, args, options);
 		const parseResult = await parseReportOutput(stdout);
 		if (parseResult === undefined) {
 			return { statusCode: 2, result: undefined, error: stderr };
 		}
 		console.error(`stderr: ${stderr}`);
 
 		return { statusCode: 0, result: parseResult };
-	} catch (error) {
+	}
+	catch (error) {
```

Comment:
Please check if this change is intentional (not sure if it was made by `prettier`)